src/R/Support/Impl/Help/IntelliSenseRHost.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Common.Core.Shell;
+using Microsoft.Languages.Editor.Shell;
+using Microsoft.R.Host.Client;
+using Microsoft.R.Support.Settings;
+
+namespace Microsoft.R.Support.Help {
+    [Export(typeof(IIntellisenseRHost))]
+    public sealed class IntelliSenseRHost : IIntellisenseRHost {
```

Comment:
`iSession.RSession` seems file. We already have such code in debugger I think.

---
In reply to: [71202975](https://github.com/Microsoft/RTVS/pull/2084#discussion_r71202975) [](ancestors = 71202975)